cockatrice/src/game/player/player.cpp

```diff
@@ -1536,6 +1543,22 @@ void Player::actMoveBottomCardToTop()
     sendGameCommand(cmd);
 }
 
+void Player::actSelectAll()
+{
+    const CardItem *card = game->getActiveCard();
+    if (!card) {
+        return;
+    }
+
+    if (const auto *zone = card->getZone()) {
+        for (auto &cardItem : zone->getCards()) {
```

Comment:
nit: Technically you should check if zone is a nullptr

Reply:
doesn't `if (const auto *zone = card->getZone())` already do that?

Reply:
I've never seen that annotation before tbf, it might?